### ISSUE TITLE
How to use multi-gpu to pre-train the model?

### ISSUE THREAD

**Issue by BinWone** [microsoft/MASS#10]
How to use multi-gpu to pre-train the model? 
must with slurm?

**Comment by StillKeepTry** [comment]
Just use `export NGPU=8; CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7 python -m torch.distributed.launch --nproc_per_node=$NGPU train.py` to run `train.py`

**Comment by BinWone** [comment]
> Just use `export NGPU=8; CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7 python -m torch.distributed.launch --nproc_per_node=$NGPU train.py` to run `train.py`

it just run the same task in 8 GPU? Can it average loss over the GPUs/nodes for every epoch to fast the   pre-training process?
![image](https://user-images.githubusercontent.com/11994164/60231625-6bbfe800-98cc-11e9-94d3-fd263ac647b0.png)

**Comment by StillKeepTry** [comment]
@BinWone Yes. Besides, you can also use --fp16 to accelerate speed if using V100.
